LICENSE.rst

```diff
@@ -1,25 +1,26 @@
-Copyright (c) 2019, SkyPy Team
-All rights reserved.
+Copyright 2020 SkyPy Team
```

Comment:
I don't think the "SkyPy Team" is formally defined anywhere. I would prefer if we used "The SkyPy Project" (i.e. our [GitHub Organization](https://github.com/skypyproject)) or "SkyPy Developers" (i.e. the [repository contributors](https://github.com/skypyproject/skypy/graphs/contributors))

Reply:
```suggestion
Copyright 2020 The SkyPy Project
```

Reply:
This would be my preference, but I think we should get some consensus from other members before making the change